### PR TITLE
dotCMS/core#24219 fix redirect from pages listing is not considering languageId

### LINKED ISSUE #24219: Favorite Pages View: Edit Mode must consider language and device
### Parent Issue

#22343

### Problem Statement

If I click on a page using the new pages list, it takes me to the Edit Mode without respecting the page language and the default device. 

### Steps to Reproduce

1. Using full starter, open any existing page in English from the new Pages view.
2. Once in Edit Mode, change the device
3. Go back to the Pages View and open a page in Spanish (for example: Destinos)
4. You should see the Edit Mode in English and the last selected device

https://user-images.githubusercontent.com/8741395/221628050-3f97a21a-dbaf-4631-8737-9f8baa03b0ce.mov

### Acceptance Criteria

When I click on any page from the list, it should behave the same way the site browser does, respecting page language and device.

### dotCMS Version

23.03

### Proposed Objective

Core Features

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
### Proposed Changes

https://user-images.githubusercontent.com/37185433/221947547-a1135b1a-0703-443d-b98f-742ecbeadeaa.mov

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Review by oidacra — APPROVED** [review]

**Review by fmontes — CHANGES_REQUESTED** [review]
.

**Review by hmoreras — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.ts [hunk 1]
```diff
@@ -61,7 +61,11 @@ export class DotPagesListingPanelComponent {
      * @memberof DotPagesListingPanelComponent
      */
     onRowSelect(event: Event): void {
-        this.goToUrl.emit(event['data'].urlMap || event['data'].url);
+        const url = `${event['data'].urlMap || event['data'].url}?language_id=${
+            event['data'].languageId
+        }&device_inode=`;
```
[the remark this answers is not in the document]
Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.ts:66]: @fmontes yes, othewise EDIT MODE will display the previous device shown, and it needs to be Desktop by default

**Review by fmontes — CHANGES_REQUESTED** [review]
N/A

**Review by zJaaal — APPROVED** [review]

**Comment by damen-dotcms** [comment]
I think desktop by default is fine. Good to move the device and language variables to the URL, but we don't need to save those as session variables. Someone edits one page, then edits another... the 2nd page would just start with "default" screen sizes and languages.

**Review by fmontes — APPROVED** [review]

**Pull request merged**